### PR TITLE
Docs: advertise links to Grafana Cloud for intro users

### PR DESCRIPTION
**Pull request by KMiller-Grafana**

### CODE REVIEW

**Review by annanay25 — APPROVED** [review]

File: docs/tempo/website/_index.md [hunk 1]
```diff
@@ -8,11 +8,13 @@ aliases:
...
-Grafana Tempo is an open source, easy-to-use and high-volume distributed tracing backend. Tempo is cost-efficient, requiring only object storage to operate, and is deeply integrated with Grafana, Prometheus, and Loki. To learn more about Grafana, refer to [What is Grafana?](https://grafana.com/docs/grafana/latest/getting-started/)
+Grafana Tempo is an open source, easy-to-use and high-volume distributed tracing backend. Tempo is cost-efficient, requiring only object storage to operate, and is deeply integrated with Grafana, Prometheus, and Loki.
+
+> **Note:** You can use [Grafana Cloud](https://grafana.com/products/cloud/features/#cloud-dashboards-grafana) to avoid installing, maintaining, and scaling your own instance of Grafana Tempo. The free forever plan includes 50GB of free traces. [Create a free account to get started](https://grafana.com/auth/sign-up/create-user?pg=docs-grafana-install&plcmt=in-text).
 
 - [Getting Started](getting-started/)
 - [Configuration](configuration/)
 - [Deployment and Operations](operations/)
 - [API](api_docs/) 
 - [Troubleshooting](troubleshooting/)
```
Comment (kvrhdn) [inline docs/tempo/website/_index.md:19]: This section should match the sidebar right? We are also missing the Grafana Agent link
```suggestion
- [API](api_docs/) 
- [Grafana Agent](grafana-agent/)
- [Troubleshooting](troubleshooting/)
```

**Review by kvrhdn — APPROVED** [review]
LGTM 🙂

File: docs/tempo/website/_index.md (hunk 1 above)
Comment (KMiller-Grafana) [reply docs/tempo/website/_index.md:19]: @kvrhdn If you have a docs issue/suggestion that has little or nothing to do with the PR, would you please open an issue or ping me outside the PR?

Comment (kvrhdn) [reply docs/tempo/website/_index.md:19]: Apologies, I thought you modified this list as well but it was only a newline being added.

**Pull request merged**